clang/lib/AST/Interp/Pointer.h

```diff
@@ -295,7 +295,7 @@ class Pointer {
   bool isUnion() const;
 
   /// Checks if the storage is extern.
-  bool isExtern() const { return Pointee->isExtern(); }
+  bool isExtern() const { return Pointee && Pointee->isExtern(); }
```

Comment:
If we need to care that `Pointee` can be null, we need to care consistently, don't we?

Reply:
Kinda, we already care in several places, but I think when interpreting we end up checking `isZero()` or `isLive()` before calling any other `Pointer` API.

Reply:
We could. Do you want me to add them?

Reply:
Yeah, I think so, just to help catch any uses that are out-of-contract.